### PR TITLE
Update api-management-service.md

### PR DESCRIPTION
**Pull request by TCP1231**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -1,5 +1,40 @@
 # Azure API Management service changelog
 
+## Release - API Management service: September 2023
+
+### New features, fixes, and improvements
+
+1. We have introduced new Azure Advisor recommendations
```
Comment (vladvino) [inline changelogs/api-management-service.md:7]: Should there be ":" at the end of this line?

File: changelogs/api-management-service.md [hunk 2]
```diff
@@ -1,5 +1,40 @@
...
+
+    - Use Azure AD authentication instead of gateway token for Configuration API
+    - Identity self-hosted gateways using gateway tokens that will expire in < 7 days
+    - Identity validate-jwt policy usage with insecure key size
+
+2. We have fixed an issue where the OpenApi would fail if the parameter names where empty.
```
Comment (vladvino) [inline changelogs/api-management-service.md:13]: Please spell OpenAPI with API in all caps.

File: changelogs/api-management-service.md [hunk 3]
```diff
@@ -1,5 +1,40 @@
...
+3. We have fixed an issue where nested objects were being overridden when import OpenApi was used.
+4. We have fixed an issue where the FixOpenApi parameter schema can now import complex schemas.
+5. We have fixed a bug where validate-azure-token policy was not properly using expressions to resolve tenant-id.
+6. Deleting API release will now return 204 instead of 404 if release does not exist.
```
Comment (vladvino) [inline changelogs/api-management-service.md:17]: Please replace "Deleting" with "We have deleted", assuming we've already done it.

File: changelogs/api-management-service.md [hunk 4]
```diff
@@ -1,5 +1,40 @@
...
+7. Removing support for the [retired set-graphql-resolver policy](https://learn.microsoft.com/en-us/azure/api-management/set-graphql-resolver-policy) which had already been replaced with GraphQL API resolvers.
```
Comment (vladvino) [inline changelogs/api-management-service.md:18]: Please replace "Removed" with "We have removed", assuming we've already done it.

File: changelogs/api-management-service.md [hunk 5]
```diff
@@ -1,5 +1,40 @@
 # Azure API Management service changelog
 
+## Release - API Management service: September 2023
```
Comment (vladvino) [inline changelogs/api-management-service.md:3]: Please group all fixes together.

**Review by vladvino — CHANGES_REQUESTED** [review]
Please see my comments. Thanks!

**Review by TCP1231 — COMMENTED** [review]
Made requested changes.

**Review by vladvino — APPROVED** [review]

**Pull request merged**